src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -49,9 +49,9 @@ private FlutterSdkUtil() {
   /**
    * Return the environment variable value to use when shelling out to the Flutter command-line tool.
    */
-  public static String getFlutterEnvValue() {
+  public static String getFlutterHostEnvValue() {
     final String clientId = ApplicationNamesInfo.getInstance().getFullProductName().replaceAll(" ", "-");
-    final String existingVar = System.getenv(FLUTTER_ENVIRONMENT);
+    final String existingVar = System.getenv(FLUTTER_HOST_ENV);
     return existingVar == null ? clientId : (existingVar + ":" + clientId);
```

Comment:
What's our allowed character set? How do we escape special characters? Maybe send JSON?